network/f5/bigip_irule_bigsuds.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+DOCUMENTATION = '''
+---
+module: bigip_irule_bigsuds
+short_description: Manage iRules on a BIG-IP.
```

Comment:
For consistence we don't put a fullstop on `short_description`,
We do however on the `description:` require them to be full sentences.